lab_1_classify_by_unigrams/main.py

```diff
@@ -84,6 +120,27 @@ def compare_profiles(
     In case of corrupt input arguments or lack of keys 'name' and
     'freq' in arguments, None is returned
     """
+    try:
+        if type(unknown_profile["name"]) == str and type(profile_to_compare["name"]) == str:
+            pass
+    except:
+        return None
+    if not (type(unknown_profile)) == dict or not (type(profile_to_compare)) == dict:
+        return None
+    else:
+        if not (type(unknown_profile["freq"])) == dict or not (type(profile_to_compare["freq"])) == dict:
+            return None
+    vocabulary = set(unknown_profile['freq'].keys()) | set(profile_to_compare['freq'].keys())
+    actual = []
+    predicted = []
+    for letter in vocabulary:
+        actual.append(unknown_profile['freq'].get(letter, 0))
+    for letter in vocabulary:
+        predicted.append(profile_to_compare['freq'].get(letter,0))
+
+    mse = calculate_mse(predicted=predicted, actual=actual)
+
+    return round(mse, 3)
```

Comment:
why 3?

Reply:
I thought we should round mse like in the example. Is it unessesary?

Reply:
You should know :)

Reply:
fixed